doc/sphinxman/source/optking.rst

```diff
@@ -41,6 +41,15 @@ Geometry Optimization
 
 *Module:* :ref:`Keywords <apdx:optking>`, `OPTKING <https://github.com/psi-rking/optking>`_
 
+.. note::
+
+   As of version `1.10`, the validation of OptKing's options has been moved from |PSIfour| into
+   OptKing. Any options can still be set directly in |PSIfour| like in previous versions; however,
+   validation of any OptKing keywords will not take place until optimize is called.
+   If an option is not available in your version of PSI4 please make sure that |PSIfour|
+   and OptKing are up-to-date. If an option is still not available, use the ``optimizer_keywords``
+   argument to :py:func:`~psi4.driver.optimize`.
```

Comment:
I like the compromise. Glad psi4 isn't preventing any optking functionality now.